build/build.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+# 
+# LiskHQ/lisk-build
+# Copyright (C) 2017 Lisk Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+cd "$(cd -P -- "$(dirname -- "$0")" && pwd -P)" || exit 2
+
+CLEAN=false
+OPTIND=1
+while getopts "cn:v:" OPT; do
+	case "$OPT" in
+		c) CLEAN=true;;
+		n) LISK_NETWORK="$OPTARG";;
+		v) VERSION="$OPTARG";;
+		:) echo 'Missing option argument for -'"$OPTARG" >&2; exit 1;;
+		*) echo 'Unimplemented option: -'"$OPTARG" >&2; exit 1;;
+	esac
+done
+
+if [[ ${VERSION:-} && ${LISK_NETWORK:-} ]]; then
+	echo "Building version $VERSION for ${LISK_NETWORK}net."
+else
+	echo "Both -n and -v are required. Exiting."
+	exit 1
+fi
+
+# shellcheck source=./config.sh
+. "$(pwd)/config.sh"
+
+if [[ "$CLEAN" == true ]]; then
+	echo "Cleaning build."
+	rm -rf "src/$BUILD_NAME"
+fi
+
+################################################################################
+
+# Requirements for Ubuntu 14.04.5 LTS base install:
+# apt-get install build-essential autoconf libtool zlib1g-dev \
+#     valgrind \  # jq tests
+#     tcl8.5 \    # redis tests
+#     moreutils   # used below
+
+pushd src
+
+if [ ! -f "$LISK_FILE" ]; then
+	echo "$LISK_FILE needs to be copied to src/"
+	exit 2
+fi
+
+echo
+echo "Downloading jq..."
+echo "--------------------------------------------------------------------------"
+[[ -f "$JQ_FILE" ]] || wget -nv "$JQ_BIN_URL" --output-document="$JQ_FILE"
+echo "$JQ_SHA256SUM  $JQ_FILE" |sha256sum -c
```

Comment:
`jq` is a dependency of the build process. Can we not just use the available jq binary and don't download new copy.